buildSrc/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ repositories {
     mavenCentral()
 }
 dependencies {
+    implementation(libs.gradle.micronaut)
     implementation(libs.spotless)
     implementation(libs.graalvm.native.buildtools)
```

Comment:
You can remove this plugin since it's applied by the `io.micronaut.library` plugin.